osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerSpunOut.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using osu.Game.Rulesets.Osu.Mods;
+
+namespace osu.Game.Rulesets.Osu.Tests
+{
+    [TestFixture]
+    public class TestSceneSpinnerSpunOut : TestSceneSpinner
+    {
+        public override IReadOnlyList<Type> RequiredTypes => base.RequiredTypes.Concat(new[] { typeof(OsuModSpunOut) }).ToList();
```

Comment:
```csharp
        public override IReadOnlyList<Type> RequiredTypes => new[]
        {
            typeof(OsuModSpunOut)
        };
```

Reply:
Unconvinced this suggestion is correct, since applying this change might not pick up types required by the base scene.